src/hooks/useScrollToEnd.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { ScrollView } from 'react-native'
+
+/**
+ * This hook will return a ref, which you can attach to a scroll view.
+ * When the `ready` parameter becomes true,
```

Comment:
Nice hook, I can see this being useful.

`ready` is pretty cryptic IMO and I'm assuming is specific to describing how the prop is currently used on its calling scene, which might not always be true in all cases. 

Let's keep it closer to what is literally happening here. How bout `triggerScrollToEnd`?

Reply:
Actually, why limit this to scrolling to the end? Let's make this: 
`useForceScrollPosition(position: undefined | 'start' | 'end')`

Reply:
Added a lil'

```ts
/** 
  * IMPORTANT: Changes in this file MUST be synced between edge-react-gui and 
  * edge-login-ui-rn! 
  */ 
```

;)

Reply:
Added a lil'

```ts
/** 
  * IMPORTANT: Changes in this file MUST be synced between edge-react-gui and 
  * edge-login-ui-rn! 
  */ 
```